### ISSUE TITLE
About real-world super-resolution

### ISSUE THREAD

**Issue by Baekhyunn** [prs-eth/thera#13]
hi，how do you make your model useful in real-world sr? I see that the low-resolution input you trained were only obtained by bicubic downsample, and did not require some complex degradation? I hope to get your answer!

**Comment by alebeck** [comment]
Hi Baekhyunn, thanks for your interest in our work! Indeed, we followed the same data generation protocol of previous ASR works and did not experiment with other degradation models on top of that. If the degradations get too different from the training data, some loss in performance should be expected. Training logic and checkpoints are however public, so if you have any further degradations in mind (e.g., JPG compression), you could easily re-train or fine-tune the existing checkpoints without modification to the architecture. Hope that helps! :)
